src/libs/ReportUtils.ts

```diff
@@ -4172,6 +4172,21 @@ function navigateToPrivateNotes(report: Report, session: Session) {
     Navigation.navigate(ROUTES.PRIVATE_NOTES_LIST.getRoute(report.reportID));
 }
 
+function getReportFieldTitle(report: OnyxEntry<Report>, reportField: PolicyReportField) {
+    const value = report?.reportFields?.[reportField.fieldID] ?? reportField.defaultValue;
+
+    if (reportField.type !== 'formula') {
+        return value;
+    }
+
+    return value.replaceAll(/{report:([a-zA-Z]+)}/g, (match, property) => {
```

Comment:
Lets add this regex here
https://github.com/Expensify/App/blob/8970aa3cd4ffc73cc59b9de3a50fd3d4da361f15/src/CONST.ts#L1342
and use it from there

Reply:
Done!